### PR TITLE
Adding speed games demo site

### PR DESCRIPTION
**Pull request by gernberg**
This adds the speed games demo site.

It is intentionally an example of "bad & inefficient code" so please no reviews on that, that is WAI.

### CODE REVIEW

File: speed_games/demo_website/public/js/third_party/jquery-1.9.1.js
Comment (MarkusBordihn) [inline speed_games/demo_website/public/js/third_party/jquery-1.9.1.js:1]: Is it possible to load jQuery over the offical jQuery CDN like https://releases.jquery.com/ like the other files?

If this needs to be included as local file, it should be at least added over npm or as a git-sub-module to avoid having it as part of the code base. Happy to help with this.

File: speed_games/demo_website/package.json
Comment (MarkusBordihn) [inline speed_games/demo_website/package.json:2]: Looks like a template placeholder, if possible please adjust the name.

File: speed_games/demo_website/package.json
Comment (MarkusBordihn) [inline speed_games/demo_website/package.json:20]: Looks like a template placeholder, if possible please adjust the name or remove it, if not needed.

**Review by MarkusBordihn — APPROVED** [review]
Hi Gustav,
I pre-approve the PR, please review the comments before merging these changes. Thanks.

File: speed_games/demo_website/public/js/third_party/jquery-1.9.1.js
Comment (gernberg) [reply speed_games/demo_website/public/js/third_party/jquery-1.9.1.js:1]: For the purpose of the demo site, I prefer to leave it like this.

It's meant to demonstrate the impact of render blocking resources, and the extreme impact they can have when left unminified & uncompressed.
The public CDN doesn't offer this option, and I feel with npm it's a bit too hidden for the purpose of what I want to teach with this exercise.
For just one file a git sub-repo feels a bit overkill, but I might misunderstand something here.

(and technically yes agree, this is bad practice but that's the point of the demo site)

Comment (gernberg) [reply speed_games/demo_website/public/js/third_party/jquery-1.9.1.js:1]: > Hi Gustav, I pre-approve the PR, please review the comments before merging these changes. Thanks.

Thanks, btw - pre-approving doesn't seem to work for GitHub repos since the approve becomes outdated when I push a new commit :)

**Review by MarkusBordihn — APPROVED** [review]

**Pull request merged**
